.golangci.yml

```diff
@@ -68,9 +68,9 @@ issues:
     # If future reflect.Kinds are nil-able, we'll find out when a test fails.
     - linters: [exhaustive]
       path: internal/assert/assert.go
-    # We need our duplex HTTP call to have access to the context.
-    - linters: [containedctx]
-      path: duplex_http_call.go
+    # We need our HTTP call to have access to the context and ignore body close.
+    - linters: [containedctx, bodyclose]
+      path: http_call.go
```

Comment:
Why exactly do we now ignore closing the body? Superficially, this doesn't seem right.

Also, I think it would be better to add these exclusions via `nolint` comments, in the actual code, to make it much easier to assess and understand the lint exclusion alongside, by examining the corresponding code.

Reply:
I thought it was desired to use golangci over nolint.